server/src/main/java/org/apache/iotdb/db/engine/storagegroup/StorageGroupProcessor.java

```diff
@@ -401,7 +415,13 @@ public void insert(InsertPlan insertPlan) throws QueryProcessException {
       Integer[] results = new Integer[batchInsertPlan.getRowCount()];
       long lastFlushTime = latestFlushedTimeForEachDevice.get(batchInsertPlan.getDeviceId());
 
+      /*
+       * sequence time range id -> indexes indicate which data should insert
```

Comment:
To be more precise:
indexes indicate which data should insert -> indexes indicating which rows should be inserted

Reply:
fixed~